backend/globaleaks/handlers/base.py

```diff
@@ -250,7 +255,8 @@ def validate_request(request, request_template):
 
                 if not BaseHandler.validate_type(value, request_template[key]):
                     log.err("Received key %s: type validation fail", key)
-                    raise errors.InputValidationError("Key (%s) type validation failure" % key)
+                    raise errors.InputValidationError(
+                        "Key (%s) type validation failure" % key)
```

Comment:
@joeman65 @alessiofranceschini: This seems to be just stilistic  code review, please provide this separately to not misk code review of our code with the features you need to implement.